public/style.css

```diff
@@ -182,3 +182,185 @@ input.nav_search_box{
   font-size: 24px;
   text-align: center;
 }
+
+/* These are for tye styling of the editQuiz page */
+.edit_title #edit-quiz-title-input{         
+	background-color: var(--lightgrey);         
+	text-decoration: none;         
+	font-size: 21px;         
+	font-weight: initial;     
+	color: var(--darkgrey);
+	font-style: normal; 
+	padding-bottom: 28px;
+	position: relative;
+	left: 3%;
+	width: 49%;
```

Comment:
Please don't use percents.

Reply:
how am I suppose to get the width to be the same on bigger and smaller screens then?